demo/dsa-accel-config-demo-pod.yaml

```diff
@@ -2,15 +2,36 @@ apiVersion: v1
 kind: Pod
 metadata:
   name: dsa-accel-config-demo
+  labels:
+    app: dsa-accel-config-demo
 spec:
   containers:
   - name: dsa-accel-config-demo
-    image: dsa-accel-config-demo:devel
+    image: intel/dsa-accel-config-demo:devel
+    command: [ "/bin/bash", "-e", "-c", "cd /test; ./dsa_test -v -w 0 -d dsa0/wq0.0 -f 0x0 -o 0x3; while true; do sleep 300000; done" ]
+    securityContext:
+      privileged: true
```

Comment:
some debug leftover?

Reply:
Actually no, seems like this is required at least for the bmetal. It's unclear to me if this ever worked (bisecting, cross-checking)